### PR TITLE
Allow publishing messages with an empty payload

### PR DESCRIPTION
**Pull request by madss**

### CODE REVIEW

File: aioamqp/tests/test_publish.py [hunk 1]
```diff
@@ -23,6 +23,20 @@ def test_publish(self):
...
+        yield from self.channel.queue_declare("q", exclusive=True, no_wait=False)
+        yield from self.channel.exchange_declare("e", "fanout")
+        yield from self.channel.queue_bind("q", "e", routing_key='')
+
+        # publish
+        yield from self.channel.publish("", "e", routing_key='')
+
+        queues = self.list_queues()
+        self.assertIn("q", queues)
+        self.assertEqual(1, queues["q"]["messages"])
```
Comment (dzen) [inline aioamqp/tests/test_publish.py]: we might want to test the content here ?

Comment (dzen) [reply aioamqp/tests/test_publish.py]: Imho this test should only use the publish method from this library and not depends on the other methods such as `basic_consume` to limit the module bonds. but that's a point of view.

The `message_bytes` should be enough ?

Comment (dzen) [reply aioamqp/tests/test_publish.py]: By the way, a more complete end-to-end test is defined here, if you want to see an example : https://github.com/Polyconseil/aioamqp/blob/48f0638662341af2bcf0a09613ba4b7971a2a0ee/aioamqp/tests/test_consume.py#L58

Comment (madss) [reply aioamqp/tests/test_publish.py]: Nice to know. I've amended the commit with the `message_bytes` assertion.

**Comment by dzen** [comment]
so I cherry-picked the commit, it's on master with https://github.com/Polyconseil/aioamqp/commit/45bcefc7514db5bf0fb2c5a8a670ba7dc4d43f33.

We're planning to drop both 3.3 and 3.4 support for the end of the week, we've finally drooped those versions from our codebase \o/. Expect a new release soon.

**Comment by madss** [comment]
Any update on when we can expect a release?

**Pull request closed without merging**